source/fem-dem/ib_particles_dem.cc

```diff
@@ -167,117 +164,130 @@ IBParticlesDEM<dim>::calculate_pp_contact_force(
                 }
 
               // Calculation of normal overlap
-              double normal_overlap;
+              double                  normal_overlap;
```

Comment:
I would add some comments in this section like it's done for the normal DEM. 

// i is the particle, j is the wall.
// etc. 

Every time we go back in this code, it's always hard to remember the convention that is being used.